### ISSUE TITLE
netcoreapp1.1> support

### ISSUE THREAD

**Issue by ferflores** [Antaris/RazorEngine#460]
Are there plans to make this package compatible with netcoreapp 1.1 or superior?

Nuget error: Package RazorEngine 3.9.3 is not compatible with netcoreapp1.1

**Comment by leedavi** [comment]
I think making this project compatible with to .Net Core is essential, although I understand the problems to do this might make it difficult and maybe not even possible if something is missing in .Net Core.

But I'll be interested to know if any tests have been done against .Net Core.

**Comment by john-t-white** [comment]
I agree. I would love to try this out because it has a lot of features that I need, but I have to have it support netstandard1.6

**Comment by thebentern** [comment]
Is there any timeline for when support for 1.1 will be available?

**Comment by jkheadstorm** [comment]
I too would be very interested in this for .NET Core 1.1

**Comment by jkergosi** [comment]
This seems extremely useful, but all of our development is in .NET Core. Is there a plan/ETA on this?
